lib/writer/block-write-stream.js

```diff
@@ -60,6 +61,23 @@ class BlockWriteStream extends stream.Writable {
     this.blocksRead = 0
     this.bytesWritten = 0
     this.blocksWritten = 0
+    this.meter = speedometer()
+    this.delta = 0
+    this.speed = 0
+
+    this.clear = () => {
+      clearInterval(this.timer)
+    }
+
+    this.update = () => {
+      this.speed = this.meter(this.delta)
+      this.delta = 0
+    }
+
+    this.once('end', this.clear)
+    this.once('error', this.clear)
+
+    this.timer = setInterval(this.update, 500)
```

Comment:
Should we have a named constant here e.g. `UPDATE_INTERVAL_MS` ?

Reply:
Agreed

Reply:
Added